src/components/home/questionnaire-card.tsx

```diff
@@ -0,0 +1,55 @@
+import { Typography, Card, CardContent, Grid } from "@mui/material";
+import { Link } from "react-router-dom";
+import strings from "src/localization/strings";
+import UserRoleUtils from "src/utils/user-role-utils";
+
+/**
+ * Component for displaying questionnaire card
+ */
+const QuestionnaireCard = () => {
+	const adminMode = UserRoleUtils.adminMode();
+
+  return (
+    <Link
+      to={adminMode ? "/admin/questionnaire" : "/questionnaire"}
+      style={{ textDecoration: "none" }}
+    >
+      <Card
+        sx={{
+          "&:hover": {
+            background: "#efefef",
+          },
+        }}
+      >
+        {adminMode ? (
```

Comment:
I would suggest moving conditional renders like this to their own function. Code like this can quickly become hard to read if there is multiple conditional renders.